CMakeLists.txt

```diff
@@ -727,6 +727,9 @@ elseif(WIN32)
   endif()
 endif()
 
+# We want symbols such as M_PI visible.
+add_compile_definitions(_USE_MATH_DEFINES)
```

Comment:
We use target_compile_definitions() in all other places. Perhaps do that here as well, and move this to ~line 200 where other global definitions are added.

Reply:
Done. Since I looked for `add_compile_definitions`, I didn't see the `target_compile_definitions`